Docs/beta/install/basic_install.rst

```diff
@@ -48,13 +48,13 @@ Use one of the following commands to install AIMET based on your choice of frame
 
         With CUDA 12.x:
 
-        .. parsed-literal::
+        .. code-block:: bash
```

Comment:
I think this breaks the `replace::` directives and will show the literal content of this block.
![image](https://github.com/user-attachments/assets/f9a6e488-4c05-47e7-b040-19c6960f5f5f)
(We want this to be replaced with a real url to to the whl file)

Reply:
@quic-kyunggeu OK. "parsed-literal" doesn't support the click-to-copy tool. Is that acceptable, or do we need to find a workaround so that code included like this is copyable? I think this is the only file that contained the parsed-literal directive, so it might not be hard to work around.

Reply:
For now, I think it should be fine not to support click-to-copy for this single line if it's too much extra work.
Let's revert this portion and tackle it in a separate PR